_posts/2018-04-06-all-hands.md

```diff
@@ -0,0 +1,182 @@
+  - [Review committee guidelines](https://github.com/rust-community/events-team/pull/20)
+  - Communication and transparency (e.g. [timelines](https://github.com/rust-community/events-team/pull/22))
+  - Attendee experience, accessibility and support
+
+### Documentation
+
+- **Edition planning**: determined resources needed for the 2018 edition, what
+  that means for the Rust Bookshelf, and who will be responsible for each.
+- **Team blog**: “This week in Rust docs” is going to move to a new [docs team blog](https://rust-docs.github.io/blog/)!
+- **Doxidize** (aka rustdoc2): made [initial public release](https://github.com/steveklabnik/doxidize); it's like https://docusaurus.io/ but for Rust.
+- **Intermediate-level docs**: contributed to [idea generation](https://github.com/rust-docs/team/issues/8).
+
+### Tools
+
+- **Edition planning**, especially for the rustfix tool.
+- **Clippy lint audit**: developed plan for reaching 1.0 on Clippy this year.
+- **Custom test frameworks**: reached consensus on most of the details for [the RFC](https://github.com/rust-lang/rfcs/pull/2318)
+- **IDEs**: discussed improvements to code completion, stability
+  improvements, and new features like refactoring and auto-import support.
+
+### Cargo
+
+- **Xargo integration**: making a few more platforms tier 1 addresses the
+  immediate need for embedded; otherwise, plan to go
+  the
+  ["std-aware Cargo" route](https://github.com/rust-lang/rfcs/pull/1133#issuecomment-362355002) late
+  in 2018. Key insight: will entirely obsolete the concept of "targets" in rustup.
+- **Rustup integration**: with Xargo integration we can simplify rustup; plan to
+  expose new interface via Cargo late in 2018.
+- **Custom registries**: ready to stabilize.
+- **Profiles**: the v2 design is clear, and **Aleksey Kaldov plans to implement**.
+- **Public dependencies**: significantly revised plan to have more conservative ecosystem impact. **Aaron Turon will write a blog post**.
+- **Build system integration**: determined two pieces of functionality to implement to decouple the RLS from Cargo.
+- **Project templates**: developed a minimal design; **withoutboats will write an RFC**.
+- **Custom workflows**: designed workflow customization, which is useful for frameworks; **Aaron Turon will write an  RFC**.
+
+### Infrastructure
+
+- **bors queue**: hatched and resourced lots of ideas to reduce the PR queue for Rust 2018.
+- **crater**: pietroalbini is testing a bot for running crater!
+- **Travis log bot**: TimNN has written a bot to [extract errors] from Travis logs
+
+[extract errors]: https://github.com/rust-lang/rust/pull/49513#issuecomment-377538323
+
+### WG: CLI apps
+
+- [WG overview slides](https://git.io/rust-all-hands-cli).
+- **Survey and strategy**: dove into survey data and developed strategy from it; posts forthcoming.
+- **Distribution**: "distribution-friendly" badge on crates.io; [extensible Carogo install RFC](https://github.com/rust-lang/rfcs/pull/2376).
```

Comment:
this might read better as 2 bullet points

Reply:
also, *cargo